aten/src/ATen/native/mps/operations/Convolution.mm

```diff
@@ -47,6 +47,8 @@ Tensor _mps_convolution(
     IntArrayRef stride,
     IntArrayRef dilation,
     int64_t groups) {
+  TORCH_CHECK(input_t.dim() != 5, "Conv3D is not supported on MPS");
```

Comment:
Shouldn't this be `<5` ?
Also should we check lower limits as well?

Reply:
Good catch.

Reply:
Lower limits of 1D and 2D are supported, so we should be fine. For 3D we want a new Kernel.

Reply:
Ok, so we want to check that this is 3 or 4D.